### PR TITLE
Introduce new MIN 9 RN

### PR DESCRIPTION
**Pull request by ConnorLand**
Creating this PR for Stefanos and I to improve the Make it Native RN

### CODE REVIEW

File: content/refguide/getting-the-make-it-native-app.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ tags: ["Android", "iOS", "native", "debug"]
 
 ## 1 Introduction
 
-The Make It Native app allows developers to preview, test, and debug native mobile app projects in conjunction with Mendix Studio Pro. It is available for both Android and iOS devices.
+The Make It Native app allows developers to preview, test, and debug native mobile app projects in conjunction with Mendix Studio Pro. It currently comes in two versions: the **Make It Native** app for Studio Pro v8.8 and below, and the **Make It Native 9 app** for Studio Pro v9.0 and above. Both of these apps are available for both Android and iOS devices.
```
Comment (ConnorLand) [inline content/refguide/getting-the-make-it-native-app.md:10]: @stmarkidis Below these changes you'll see the QR codes and other offerings for getting the Make It Native app. Can you please update these sections with other QR codes for the Make It Native 9 app?

File: content/releasenotes/mobile/make-it-native-9.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+menu_order: 9
+description: "These release notes showcase each release of the iOS and Android Make It Native app versions."
+tags: ["Native", "Android", "iOS", "Apple", "Android"]
+---
+
+For more information on the Make It Native app, see [Getting the Make It Native App](/refguide/getting-the-make-it-native-app).
+
+You can download the Make It Native app for [iOS](https://apps.apple.com/app/make-it-native/id1334081181) or [Android](/refguide/getting-the-make-it-native-app).
+
+## Android X.X.X / iOS X.X.X
```
Comment (ConnorLand) [inline content/releasenotes/mobile/make-it-native-9.md:13]: @smarki Please update this dummy section with the actual RN for all editions of the Make It Native 9 app.

File: content/refguide/getting-the-make-it-native-app.md (hunk 1 above)
Comment (smarki) [reply content/refguide/getting-the-make-it-native-app.md:10]: 3 versions; The legacy one that we only serve via the qr codes; MiN 8 which is released on the Stores for latest Mendix 8 and MiN 9 for Mendix 9

File: content/releasenotes/mobile/make-it-native-9.md (hunk 2 above)
Comment (smarki) [reply content/releasenotes/mobile/make-it-native-9.md:13]: Yep will do

**Pull request merged**